jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -580,6 +581,43 @@ public void setFrustumTop(float frustumTop) {
         onFrustumChange();
     }
 
+    /**
+     * Obtains field of view when the camera is in perspective mode.
+     *
+     * @return Frame of view angle along the Y in degrees, or -1 if the camera is in orthogonal mode.
+     */
+    public float getFov() {
+        return !this.parallelProjection ? currentFov : -1;
```

Comment:
I think an orthogonal camera should return zero, not -1.

Reply:
Perhaps. I picked -1 because that is my goto "invalid" value for numbers. However, since 0 isn't a valid FOV either, that could work, too.